dbdialog/src/main/java/org/pentaho/ui/util/Launch.java

```diff
@@ -32,7 +32,7 @@ public enum Status {
     Success, Failed
   }
 
-  public static Status openURL( String url ) {
```

Comment:
why this change?

Reply:
to allow for overriding it in a test, it keeps opening a browser window during tests otherwise

Reply:
👍